packages/fluentui/react-northstar/test/specs/components/Form/FormButton-test.tsx

```diff
@@ -8,5 +8,7 @@ describe('FormButton', () => {
     constructorName: 'FormButton',
     forwardsRefTo: `Button`,
     targetComponent: Button,
+    getTargetElement: (result, attr) =>
+      attr === 'className' ? result.container.querySelector(`.${formButtonClassName}`) : result.getByRole('button'),
```

Comment:
Since react-conformance no longer uses enzyme's `find` method to look for an instance of a particular component, we have to look for where attributes are applied in the DOM instead. Open to suggestions from the northstar team if there's a more appropriate way to do this.